LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 UP42
+Copyright (c) 2020
```

Comment:
I think you should add your name in order to hold the copyright. This license protects you from any kind of liability, and credit should be given to who owes it :)

Reply:
Thank you @jmigueldelgado